AK/NumberFormat.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2020-2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "AK/Assertions.h"
+#include "AK/DeprecatedString.h"
+#include <AK/NumberFormat.h>
+#include <AK/StringView.h>
+
+namespace AK {
+
+// FIXME: Remove this hackery once printf() supports floats.
+static DeprecatedString number_string_with_one_decimal(u64 number, u64 unit, StringView suffix)
+{
+    int decimal = (number % unit) * 10 / unit;
```

Comment:
I think `int` might not be safe here. `number` and `unit` are both `u64`. Perhaps this should be `u64` or just simply `auto`?

Reply:
The function previously existed for a while, I just moved it out of line.

For being unsafe: The value is of type `u64` until it is assigned to the variable. At that point it is cast to an `int`. The value being cast to int is always less than 10. Still, just marking it as `auto` would be more consistent with the style. I'll change it to that :)